### ISSUE TITLE
Add a sample view for HIG > Designing effective charts > continuity among multiple charts (picker accessibility label)

### ISSUE THREAD

**Issue by S-Shimotori** [S-Shimotori/SampleViewer#90]
# Background

#40 

I set `accessibilityLabel` with #88, but it doesn't work.
VoiceOver read texts referred by `Text("")`, not `.accessibilityLabel(Text(""))`.

https://github.com/S-Shimotori/SampleViewer/blob/535237a38d0a93c3412a50cba2bc13f834593d9d/SampleViewer/View/ChartTrendDetailView.swift#L63

# TODO

- Configure the accessibility label correctly

||daily|weekly|monthly|half-yearly|yearly|
|---|---|---|---|---|---|
|enUS|D "Day"|W "Week"|M "Month"|6 M "6 months"|Y "Year"|
|jaJP|日 "ひ"|週 "しゅう"|月 "つき"|6か月 "ろっかげつ"|年 "とし"|

**Comment by S-Shimotori** [comment]
||enUS|jaJP|
|---|---|---|
|D|<img width="250" alt="Screenshot 2024-12-25 at 13 18 45" src="https://github.com/user-attachments/assets/81ea6bea-1153-4f9a-b805-7e2898fee3ab" />|<img width="250" alt="Screenshot 2024-12-25 at 13 25 42" src="https://github.com/user-attachments/assets/673a2622-90c6-457f-8add-7253dfc174bf" />|
|W|<img width="250" alt="Screenshot 2024-12-25 at 13 19 21" src="https://github.com/user-attachments/assets/701c80e6-89c4-41cf-b268-600edd2a2000" />|<img width="250" alt="Screenshot 2024-12-25 at 13 25 48" src="https://github.com/user-attachments/assets/bf43fd2a-b7b6-4d6c-9782-05fc7cf6fb8d" />|
|M|<img width="250" alt="Screenshot 2024-12-25 at 13 19 37" src="https://github.com/user-attachments/assets/a1d1e8b2-10aa-40e5-8a08-4134399cc676" />|<img width="250" alt="Screenshot 2024-12-25 at 13 25 56" src="https://github.com/user-attachments/assets/12418b27-ced9-4a93-8270-3f4e98a37a27" />|
|6 M|<img width="250" alt="Screenshot 2024-12-25 at 13 19 50" src="https://github.com/user-attachments/assets/a9d841ef-0f31-4a1b-bd33-76a41ac6a354" />|<img width="250" alt="Screenshot 2024-12-25 at 13 26 03" src="https://github.com/user-attachments/assets/2d487fac-e0de-4138-8540-e990541a360c" /><br>"ろっけつき"|
|Y|<img width="250" alt="Screenshot 2024-12-25 at 13 20 07" src="https://github.com/user-attachments/assets/3620d5d2-ac15-45b7-b88d-64e14a2982fc" />|<img width="250" alt="Screenshot 2024-12-25 at 13 26 12" src="https://github.com/user-attachments/assets/da192278-fafd-4120-9fb7-1346021467cd" />|
